pages/maps/MapChart.js

```diff
@@ -27,47 +29,67 @@ const offsets = {
 const MapChart = () => {
   return (
     <ComposableMap projection="geoAlbersUsa">
-      <Geographies geography={geoUrl}>
-        {({ geographies }) => (
-          <>
-            {geographies.map((geo) => (
-              <Geography
-                key={geo.rsmKey}
-                stroke="#FFF"
-                geography={geo}
-                fill="#DDD"
-              />
-            ))}
-            {geographies.map((geo) => {
-              const centroid = geoCentroid(geo);
-              const cur = allStates.find((s) => s.val === geo.id);
-              return (
-                <g key={geo.rsmKey + '-name'}>
-                  {cur &&
-                    centroid[0] > -160 &&
-                    centroid[0] < -67 &&
-                    (Object.keys(offsets).indexOf(cur.id) === -1 ? (
-                      <Marker coordinates={centroid}>
-                        <text y="2" fontSize={14} textAnchor="middle">
-                          {cur.id}
-                        </text>
-                      </Marker>
-                    ) : (
-                      <Annotation
-                        subject={centroid}
-                        dx={offsets[cur.id][0]}
-                        dy={offsets[cur.id][1]}>
-                        <text x={4} fontSize={14} alignmentBaseline="middle">
-                          {cur.id}
-                        </text>
-                      </Annotation>
-                    ))}
-                </g>
-              );
-            })}
-          </>
-        )}
-      </Geographies>
+      <ZoomableGroup>
```

Comment:
This is good but probably better to refractor and having this in a `/components/Maps/` to keep the code in this clean and easy to add more components down the line.

Reply:
Also the `data.json` should be removed and be within in the `/components/Maps/`